NOTES.WIN

```diff
@@ -89,8 +89,6 @@
 
  - UWP targets only support building the static and dynamic libraries.
 
- - The "no-uplink" must be given in the "Configure" script.
```

Comment:
I missed the bit where you disable uplink for UWP. Where is it?

Reply:
That's actually already in place:

https://github.com/openssl/openssl/blob/65dc5c3cc10af7c6f24ecd922adb7b6d17a9fe65/Configurations/50-win-onecore.conf#L83
https://github.com/openssl/openssl/blob/65dc5c3cc10af7c6f24ecd922adb7b6d17a9fe65/Configurations/50-win-onecore.conf#L92
https://github.com/openssl/openssl/blob/65dc5c3cc10af7c6f24ecd922adb7b6d17a9fe65/Configurations/50-win-onecore.conf#L101
https://github.com/openssl/openssl/blob/65dc5c3cc10af7c6f24ecd922adb7b6d17a9fe65/Configurations/50-win-onecore.conf#L110

So I really didn't have to do anything with that, just ensure that it actually works.

Reply:
Ah. Ok!